File: backup/Twitch/app/src/main/java/com/exact/twitch/ui/pagers/FollowPagerFragment.java
package com.exact.twitch.ui.pagers;

import android.os.Bundle;
import androidx.annotation.NonNull;
import androidx.annotation.Nullable;
import android.view.View;

public class FollowPagerFragment extends MediaPagerFragment {

    @Override
    public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceState) {
        super.onViewCreated(view, savedInstanceState);
        Bundle args = new Bundle(1);
        args.putString("token", "ebn5a0wxof6x770yx5r2udc0876t44"); //TODO use shared preferences
        setAdapter(new FollowPagerAdapter(requireActivity(), getChildFragmentManager(), args));
    }
}

File: app/src/main/java/com/exact/twitch/ui/pagers/FollowPagerFragment.kt
package com.exact.twitch.ui.pagers

import android.os.Bundle
import android.view.View
import androidx.core.os.bundleOf
import com.exact.twitch.util.C
import com.exact.twitch.util.TwitchApiHelper

class FollowPagerFragment : MediaPagerFragment() {

    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
        super.onViewCreated(view, savedInstanceState)
        setAdapter(FollowPagerAdapter(requireActivity(), childFragmentManager, bundleOf(C.USER to TwitchApiHelper.getUserData(requireActivity()))))
    }
}
